src/Sparse/Row.jl

```diff
@@ -419,15 +419,41 @@ end
 #  Inplace scaling
 #
 ################################################################################
+@doc raw"""
+    scale_row!(a::SRow, b::NCRingElem) -> SRow
 
+Returns the (left) product of $b \times a$.
+For Rows, the standard multiplication is from the left. 
+"""
 function scale_row!(a::SRow{T}, b::T) where T
   @assert !iszero(b)
   if isone(b)
-    return
+    return a
+  end
+  for i=1:length(a.pos)
+    a.values[i] = b*a.values[i]
+  end
+  return a
+end
+
+@doc raw"""
+  scale_row_right!(a::SRow, b::NCRingElem) -> SRow
```

Comment:
```suggestion
    scale_row_right!(a::SRow, b::NCRingElem) -> SRow
```
There must always be exactly four spaces for the indentation here